packages/clay-localized-input/src/index.tsx

```diff
@@ -111,6 +115,36 @@ const LocalizedInput = React.forwardRef<HTMLInputElement, IProps>(
 
 		const defaultLanguage = locales[0]!;
 
+		const languagePickerLocales = useMemo(() => {
+			locales.forEach((locale, index) => {
+				if (!locale.id) {
+					locales[index]!['id'] = locale.label;
+				}
+			});
+
+			return locales as Array<Item>;
+		}, []);
+
+		let languagePickerTranslations: any = {};
+
+		languagePickerTranslations = useMemo(() => {
+			Object.keys(translations).forEach((locale) => {
+				languagePickerTranslations[locale] = {
+					total: 1,
+					translated: 1,
+				};
+			});
+
+			return languagePickerTranslations;
+		}, []);
+
+		useMemo(() => {
+			languagePickerTranslations[selectedLocale?.id!] = {
+				total: 1,
+				translated: translations[selectedLocale.label] ? 1 : 0,
+			};
+		}, [translations]);
```

Comment:
This can all be updated to something like this:

```
const languagePickerTranslations = useMemo(() => {
	const languagePickerTranslations = {};

	locales.forEach((locale) => {
		languagePickerTranslations[locale.symbol] = {
			total: 1, // this should always be one because we just have one input being translated.
			translated: translations[locale.symbol] ? 1 : 0],
		};
	}

	return languagePickerTranslations;
}, [locales, translations]);
```

For localized input the status label should never be "translating", so make sure to check that.

Also, an empty string should be rendered as untranslated, not translated. Test this by adding a translation and then removing it.

Reply:
@ethib137 in the storybook, `translations` keys match `locales` label values. The symbol values are lower case. `translations[locale.symbol]` returns undefined because `en-us` should be `en-US` should mixed case keys still be valid or should it be an exact match?